client/food-recipes-app/src/Components/Landingpage.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import "../App.css";
+function Landingpage() {
+  const [search, setSearch] = useState("");
+
+  function searchHandler(e) {
+    setSearch(e.target.value);
+  }
+
+  function formSubmit(e) {
+    e.preventDefault();
+  }
+
+  return (
+    <nav className="navbar">
```

Comment:
I think we can make this navbar looks more professional by using Material UI. There are quite few more things to add to this to be considered a Landing Page. I can contribute more to its look.